pacmod_msgs/msg/EngineRpt.msg

```diff
@@ -0,0 +1,8 @@
+std_msgs/Header header
+
+float64 engine_speed
+float64 engine_torque
+float64 engine_coolant_temp
```

Comment:
This value is a `uint8` in the DBC. Once this is fixed, I'm happy to approve. Everything else looks good.

Reply:
> This value is a `uint8` in the DBC. Once this is fixed, I'm happy to approve. Everything else looks good.

Sorry, make it a `int16` since it can be -40 to +215.

Reply:
Yes that's why I didn't use uint. I'll change it to int16!